code/web/src/setup/store.js

```diff
@@ -40,7 +41,7 @@ export const store = createStore(
   rootReducer,
   initialState,
 
-  compose(
+  composeWithDevTools(
```

Comment:
What's the difference between these two?

Reply:
The composeWithDevTools allows us to use the redux dev tools extension within the browser